scripts/watermarkstat

```diff
@@ -220,8 +220,12 @@ class Watermarkstat(object):
                     min_idx = element_idx
 
         if min_idx == sys.maxsize:
-            print("Object map is empty!", file=sys.stderr)
-            sys.exit(1)
+            if key not in ['q_shared_multi', 'q_shared_all']:
+                print("Object map is empty!", file=sys.stderr)
+                sys.exit(1)
+            else:
+                print("Object map from COUNTERS_DB is empty, because counters are not configured in CONFIG_DB!")
```

Comment:
because counters are not configured in CONFIG_DB! -> because multicast queues are not configured in CONFIG_DB